lib/utility/src/WatchdogWrapper.cpp

```diff
@@ -14,6 +18,14 @@ void WatchdogWrapper::startWatchdog(std::chrono::milliseconds countdown_ms /*= 5
   pet_thread.start(callback(WatchdogWrapper::petWatchdog, &pet_ms));
 }
 
+void WatchdogWrapper::logResetReason() {
```

Comment:
This can be changed later to a function that can log all reasons for reset (Power On, Software Reset, Hardware Reset), but for now it just logs a reset and timestamp if it was triggered by the watchdog timer